pkg/apitagging/filters.go

```diff
@@ -31,21 +32,34 @@ type serviceFilter struct {
 var serviceFilters = map[string]serviceFilter{
 	"AWS/ApiGateway": {
 		FilterFunc: func(ctx context.Context, client Client, inputResources []*model.TaggedResource) ([]*model.TaggedResource, error) {
-			promutil.APIGatewayAPICounter.Inc()
-			var limit int64 = 500 // max number of results per page. default=25, max=500
 			const maxPages = 10
-			input := apigateway.GetRestApisInput{Limit: &limit}
-			output := apigateway.GetRestApisOutput{}
-			var pageNum int
-			var outputResources []*model.TaggedResource
+
+			var (
+				limit           int64 = 500 // max number of results per page. default=25, max=500
+				input                 = apigateway.GetRestApisInput{Limit: &limit}
+				output                = apigateway.GetRestApisOutput{}
+				pageNum         int
+				outputResources []*model.TaggedResource
+			)
+
 			err := client.apiGatewayAPI.GetRestApisPagesWithContext(ctx, &input, func(page *apigateway.GetRestApisOutput, lastPage bool) bool {
+				promutil.APIGatewayAPICounter.Inc()
```

Comment:
I noticed we were incrementing only once per every method call, but I think we want to track the actual number of API calls, so I moved this line to increment on every call.